extensions/wikia/DesignSystem/scripts/DesignSystemDropdowns.js

```diff
@@ -0,0 +1,22 @@
+$(function ($) {
+	'use strict';
+
+	$('body').on('click', function(event) {
+		var eventTarget = $(event.target),
```

Comment:
These variables should be prefixed with `$` as they are jQuery objects per conventions.